src/mono/mono/metadata/bundled-resources-internals.h

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+#ifndef __MONO_METADATA_BUNDLED_RESOURCES_INTERNALS_H__
+#define __MONO_METADATA_BUNDLED_RESOURCES_INTERNALS_H__
+
+#include "loader-internals.h"
+
+#include <mono/metadata/assembly.h>
+#include <mono/metadata/mono-private-unstable.h>
+
+typedef enum {
+	MONO_BUNDLED_DATA,
+	MONO_BUNDLED_ASSEMBLY,
+	MONO_BUNDLED_SATELLITE_ASSEMBLY,
+	MONO_BUNDLED_RESOURCE_COUNT,
+} MonoBundledResourceType;
+
+typedef struct _MonoBundledResource {
+	MonoBundledResourceType type;
+	void (*free_bundled_resource_func)(void *);
+} MonoBundledResource;
+
+typedef struct _MonoBundledData {
+	char *name;
+	const unsigned char *data;
+	unsigned int size;
+} MonoBundledData;
+
+typedef struct _MonoBundledDataResource {
+	MonoBundledResource resource;
+	MonoBundledData data;
+} MonoBundledDataResource;
+
+typedef struct _MonoBundledSymfile {
```

Comment:
Drop file from name, all resource normally originate from files, so nothing special about this one, other structs that just describe data and not a full resource is suffixed with Data, so maybe we should call this MonoBundledSymbolData?

Reply:
Changed, I was mainly maintaining the same naming as done in `mono-debug.c`, but you're right, the data held here isnt the `symfile` itself, just the symbol data we care about. Theres going to be some confusion between the "symbol" representing a resource "name" and the "symbol" to represent the assembly/satellite assembly symbol data in EmitBundle. I'll clean that up more